src/internal/header.rs

```diff
@@ -1,25 +1,22 @@
 //! RPM package headers
 
 use super::{tag::Tag, td::TagData};
-use crate::{License, Package, Version};
+use crate::Package;
 use std::mem;
 
 /// RPM package header
 pub(crate) struct Header(*mut librpm_sys::headerToken_s);
 
 impl Header {
-    /// Create a new header from an `librpm_sys::Header`.
-    pub(crate) fn new(ffi_header: librpm_sys::Header) -> Self {
-        unsafe {
-            // Increment librpm's internal reference count for this header
-            librpm_sys::headerLink(ffi_header);
-        }
-
+    pub(crate) unsafe fn from_ptr(ffi_header: librpm_sys::Header) -> Self {
```

Comment:
For bonus points, it would be good to add a `/// SAFETY: ...` comment above here.

Unfortunately the original (which I wrote) lacked one, or even an `unsafe fn`, oops. So consider it optional.

Generally it'd probably be a good idea to use `clippy` to lint for the presence of `/// SAFETY: ...` comments on all `unsafe` usage. I'd be happy to do that in a follow-up PR, especially as it'd give me an opportunity to brain dump into the comments what I remember of how the various `unsafe` code hangs together.